modules/Kconfig.syst

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2019 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config MIPI_SYST_LIB
+	bool "MIPI SyS-T Library Support"
+	select NEWLIB_LIBC
```

Comment:
why do you need newlib?
Also, move this to the same commit where you introduce the module in west.yml.

Reply:
sys-t lib requires some header files which can't be founded in Minimal libc.